src/main/java/seedu/application/logic/commands/RemindCommand.java

```diff
@@ -0,0 +1,42 @@
+package seedu.application.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import seedu.application.model.Model;
+import seedu.application.model.application.UpcomingInterviewPredicate;
+
+/**
+ * Reminds users of upcoming interviews within 1 week from now stored in CinternS.
+ */
+public class RemindCommand extends Command {
+
+    public static final String COMMAND_WORD = "remind";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Shows all upcoming interviews within 1 week from "
+            + "now stored in CinternS.\n"
+            + "Example: " + COMMAND_WORD;
+
+    public static final String SHOWING_REMIND_MESSAGE = "Opened reminder window.";
+
+    private final UpcomingInterviewPredicate upcomingInterviewPredicate;
+
+    /**
+     * Constructs RemindCommand object.
+     */
+    public RemindCommand() {
+        this.upcomingInterviewPredicate = new UpcomingInterviewPredicate();
+    }
```

Comment:
I was wondering if maybe instead of creating an UpcomingInterviewPredicate for each RemindCommand, we could have just one shared static instance, since the predicate is always the same anyway?
```suggestion
    private static final UpcomingInterviewPredicate UPCOMING_INTERVIEW_PREDICATE;
```